Gemfile

```diff
@@ -0,0 +1,10 @@
+source "http://rubygems.org"
+
+group :development do
+  gem 'therubyracer'
+  gem 'jekyll'
+  gem 'kramdown'
+  gem 'pygments.rb'
+  gem 'rake'
+  gem 'html-proofer'
+end
```

Comment:
Travis needs the Gemfile to execute deploy.rb in the context of bundler, otherwise it fails.  Why wouldn't it make sense to have a Gemfile here if deploy.rb relies on gems?